### PR TITLE
add remaining metadata

### PR DESCRIPTION
**Pull request by elharo**
@loosebazooka @patflynn @briandealwis

### CODE REVIEW

**Comment by elharo** [comment]
Ping @loosebazooka @patflynn @briandealwis

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 1]
```diff
@@ -1,5 +1,105 @@
...
+  },
+  {
+    "name": "Google Cloud Natural Language",
+    "documentation": "https://cloud.google.com/language/docs/reference/libraries",
+    "description":"The Google Cloud Natural Language API provides natural language understanding technologies including sentiment analysis, entity analysis, and syntax analysis.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/language.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Natural Language",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Ditto _Java Client_

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 2]
```diff
@@ -1,5 +1,105 @@
...
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/spanner.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Spanner",
+      "language":"java",
+      "site":"https://cloud.google.com/spanner/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/spanner/package-summary.html",
+      "infotip":"The com.google.cloud.spanner packages",
+      "launchStage" : "beta",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: I personally find _launchStage_ a confusing keyword.

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 3]
```diff
@@ -1,5 +1,105 @@
...
+  },
+  {
+    "name": "Google Cloud Pub/Sub",
+    "documentation": "https://cloud.google.com/pubsub/docs/reference/libraries",
+    "description":"Google Cloud Pub/Sub provides reliable, many-to-many, asynchronous messaging between applications. Publisher applications can send messages to a topic and other applications can subscribe to that topic to receive the messages.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/pubsub.svg",
+    "clients": [
+      {
+      "name":"Google Cloud Java Client for Pub/Sub",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Ditto _Java Client_

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 4]
```diff
@@ -1,5 +1,105 @@
...
+      "language":"java",
+      "site":"https://cloud.google.com/pubsub/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/latest/apidocs/com/google/cloud/pubsub/package-summary.html",
+      "infotip":"The com.google.cloud.pubsub.v1 packages",
+      "launchStage" : "beta",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Could you make the spacing before and after `:` consistent?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 5]
```diff
@@ -73,5 +173,55 @@
...
+  },
+  {
+    "name": "Stackdriver Logging",
+    "documentation": "https://cloud.google.com/logging/docs/reference/libraries",
+    "description":"Stackdriver Logging enables you to store, search, analyze, monitor, and alert on log data and events from Google Cloud Platform and Amazon Web Services in real-time.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/logging.svg",
+    "clients": [
+      {
+      "name":"Idiomatic Java Client for Stackdriver Logging",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Ditto _Java Client_

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 6]
```diff
@@ -1,5 +1,105 @@
 [
   {
+    "name": "Google Cloud Vision",
+    "documentation": "https://cloud.google.com/vision/docs/reference/libraries",
+    "description":"The Google Cloud Vision API provides vision features including image labeling, face and landmark detection, optical character recognition (OCR), and tagging of explicit content.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/vision.svg",
+    "clients": [
+      {
+      "name":"Google Java Client for Cloud Vision",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: _Java Client_ seems odd.  _Java Client Libraries for Google Cloud Vision_?  _Google Cloud Vision Client Libraries for Java_?  The libraries pages themselves use _"Cloud Vision API Client Libraries"_.

Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: Java Client Libraries for the Google Cloud Vision API

File: src/main/java/com/google/cloud/tools/libraries/libraries.json (hunk 1 above)
Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: Java Client Libraries for the Google Cloud Natural Language API

File: src/main/java/com/google/cloud/tools/libraries/libraries.json (hunk 2 above)
Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: I think we picked this at @paflynn's suggestion for compatibility with other docs.

File: src/main/java/com/google/cloud/tools/libraries/libraries.json (hunk 3 above)
Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: Java Client Libraries for the Google Cloud Pub/Sub API

File: src/main/java/com/google/cloud/tools/libraries/libraries.json (hunk 4 above)
Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: Sure. Is one way or the other more common in JSON?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json (hunk 5 above)
Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: Java Client Libraries for the Stackdriver Logging API.

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 7]
```diff
@@ -1,5 +1,30 @@
 [
   {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Compute Engine?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 8]
```diff
@@ -1,5 +1,30 @@
 [
   {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: This icon doesn't exist, and actually none  of the URLs seem correct.  Perhaps the tests should check the URLs for 404s?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 9]
```diff
@@ -1,5 +1,30 @@
...
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
+      "site":"https://cloud.google.com/compute/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/dlp/v2beta1/apidocs/com/google/cloud/compute/package-summary.html",
+      "infotip":"The com.google.cloud.dlp.v2beta1 packages",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Editorial aside: I really don't like these package names: it makes updating to a new version painful even when there is no API breakage

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 10]
```diff
@@ -1,5 +1,30 @@
...
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
+      "site":"https://cloud.google.com/compute/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/dlp/v2beta1/apidocs/com/google/cloud/compute/package-summary.html",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: This URL doesn't work for me.  This one does: https://googlecloudplatform.github.io/google-cloud-java/0.23.0/apidocs/com/google/cloud/dlp/v2beta1/package-summary.html

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 11]
```diff
@@ -1,5 +1,30 @@
...
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/dlp/compute.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
+      "site":"https://cloud.google.com/compute/docs/reference/libraries#client-libraries-install-java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: https://cloud.google.com/dlp/docs/reference/libraries#client-libraries-install-java

Comment (elharo) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: These don't need to be checked  manually if anyone will approve https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/448

Comment (briandealwis) [reply src/main/java/com/google/cloud/tools/libraries/libraries.json]: That won't pick up links for a different component though :-)

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 12]
```diff
@@ -1,5 +1,325 @@
...
+    "name": "Google Cloud Resource Manager",
+    "documentation": "https://cloud.google.com/resource-manager/docs/reference/libraries",
+    "description":"The Google Cloud Resource Manager service enables you to programmatically manage hierarchical resource containers such as organizations, folders, and projects.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/resourcemanager/resourcemanager.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Resource Manager API",
+      "compute":"java",
+      "site":"https://cloud.google.com/speech/docs/reference/libraries#client-libraries-install-java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: This URL is referencing `speech`

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 13]
```diff
@@ -1,5 +1,325 @@
...
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/resourcemanager/apidocs/com/google/cloud/compute/package-summary.html",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: This URL doesn't look right: it's referencing `compute`

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 14]
```diff
@@ -1,5 +1,325 @@
...
+      "infotip":"The com.google.cloud.speech.resourcemanager packages",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: Referencing `speech`

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 15]
```diff
@@ -1,5 +1,325 @@
...
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/dlp.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Data Loss Prevention API",
+      "compute":"java",
+      "site":"https://cloud.google.com/dlp/docs/reference/libraries#client-libraries-install-java",
+      "apireference":"https://googlecloudplatform.github.io/google-cloud-java/dlp/v2beta1/apidocs/com/google/cloud/compute/package-summary.html",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: This isn't right: references `compute`

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 16]
```diff
@@ -1,5 +1,325 @@
...
+       }
+      }
+    ]
+  },
+  {
+    "name": "Video Intelligence",
+    "documentation": "https://cloud.google.com/video-intelligence/docs/",
+    "description":"The Google Cloud Video Intelligence API enables developers to annotate videos stored in Google Cloud Storage with video and frame-level (1 fps) contextual information. The API supports common video formats including .mov, .mp4, and .avi.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/video-intelligence.svg",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: 404.  We sure could use a consistent placement of logos!  This URL works: https://cloud.google.com/images/products/video-intelligence/video-intelligence-icon.svg  (link snarfed from https://cloud.google.com/products/machine-learning/)

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 17]
```diff
@@ -1,5 +1,325 @@
...
+       }
+      }
+    ]
+  },
+   {
+    "name": "Stackdriver Monitoring",
+    "documentation": "https://cloud.google.com/monitoring/docs/reference/libraries",
+    "description":"Stackdriver Monitoring provides visibility into the performance, uptime, and overall health of cloud-powered applications.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/monitoring.svg",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: 404, but I didn't find anything obvious

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 18]
```diff
@@ -1,5 +1,325 @@
...
+       }
+      }
+    ]
+  },
+ {
+    "name": "Stackdriver Error Reporting",
+    "documentation": "https://cloud.google.com/errorreporting/docs/reference/libraries",
+    "description":"Stackdriver Error Reporting counts, analyzes and aggregates the crashes in your running cloud services.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/errorreporting.svg",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: 404, but I didn't see anything obvious either.

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 19]
```diff
@@ -1,5 +1,325 @@
...
+ {
+    "name": "Cloud DNS",
+    "documentation": "https://cloud.google.com/dns/docs/reference/libraries",
+    "description":"Google Cloud DNS is a managed authoritative Domain Name System (DNS) service running on Google infrastructure. Cloud DNS translates domain names such as www.google.com into IP addresses such as 74.125.29.101.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/dns.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google dns Engine API",
+      "dns":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"dns"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 20]
```diff
@@ -1,5 +1,325 @@
...
+  {
+    "name": "Google Cloud Vision",
+    "documentation": "https://cloud.google.com/vision/docs/reference/libraries",
+    "description":"The Google Cloud Vision API provides vision features including image labeling, face and landmark detection, optical character recognition (OCR), and tagging of explicit content.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/vision.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Vision API",
+      "vision":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"vision"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 21]
```diff
@@ -1,5 +1,325 @@
...
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Vision",
+    "documentation": "https://cloud.google.com/vision/docs/reference/libraries",
+    "description":"The Google Cloud Vision API provides vision features including image labeling, face and landmark detection, optical character recognition (OCR), and tagging of explicit content.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/vision.svg",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: 404s

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 22]
```diff
@@ -1,5 +1,325 @@
...
   {
+    "name": "Google Cloud Speech",
+    "documentation": "https://cloud.google.com/speech/docs/reference/libraries",
+    "description":"Google Cloud Speech API converts audio to text in over 100 languages.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/speech.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Speech API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 23]
```diff
@@ -1,5 +1,325 @@
...
+  {
+    "name": "Google Cloud Resource Manager",
+    "documentation": "https://cloud.google.com/resource-manager/docs/reference/libraries",
+    "description":"The Google Cloud Resource Manager service enables you to programmatically manage hierarchical resource containers such as organizations, folders, and projects.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/resourcemanager/resourcemanager.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Cloud Resource Manager API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 24]
```diff
@@ -1,5 +1,325 @@
...
+  },  {
+    "name": "Data Loss Prevention",
+    "documentation": "https://cloud.google.com/dlp/docs/reference/libraries",
+    "description":"The DLP API enables clients to detect the presence of Personally Identifiable Information (PII) and other privacy-sensitive data in user-supplied, unstructured data streams such as text blocks and images.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/dlp.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Data Loss Prevention API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 25]
```diff
@@ -1,5 +1,325 @@
...
+  {
+    "name": "Video Intelligence",
+    "documentation": "https://cloud.google.com/video-intelligence/docs/",
+    "description":"The Google Cloud Video Intelligence API enables developers to annotate videos stored in Google Cloud Storage with video and frame-level (1 fps) contextual information. The API supports common video formats including .mov, .mp4, and .avi.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/video-intelligence.svg",
+    "clients": [
+      {
+      "name":"A client to the Google Cloud Video Intelligence API.",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 26]
```diff
@@ -1,5 +1,325 @@
...
+  {
+    "name": "Compute Engine",
+    "documentation": "https://cloud.google.com/compute/docs/reference/libraries",
+    "description":"Google Compute Engine lets you create and run virtual machines on Google infrastructure.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/compute.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 27]
```diff
@@ -1,5 +1,325 @@
...
+   {
+    "name": "Stackdriver Monitoring",
+    "documentation": "https://cloud.google.com/monitoring/docs/reference/libraries",
+    "description":"Stackdriver Monitoring provides visibility into the performance, uptime, and overall health of cloud-powered applications.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/monitoring.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Stackdriver Monitoring API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 28]
```diff
@@ -1,5 +1,325 @@
...
+ {
+    "name": "Stackdriver Error Reporting",
+    "documentation": "https://cloud.google.com/errorreporting/docs/reference/libraries",
+    "description":"Stackdriver Error Reporting counts, analyzes and aggregates the crashes in your running cloud services.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/errorreporting.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Google Compute Engine API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 29]
```diff
@@ -1,5 +1,325 @@
...
+  {
+    "name": "Stackdriver Trace",
+    "documentation": "https://cloud.google.com/trace/docs/reference/libraries",
+    "description":"A distributed tracing system that collects latency data from App Engine applications.",
+    "transports": ["grpc"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/trace.svg",
+    "clients": [
+      {
+      "name":"Java Client Libraries for the Stackdriver Trace API",
+      "compute":"java",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: s/"compute"/"language"/ ?

File: src/main/java/com/google/cloud/tools/libraries/libraries.json [hunk 30]
```diff
@@ -69,7 +389,57 @@
...
+       }
+      }
+    ]
+  },
+  {
+    "name": "Google Cloud Translation",
+    "documentation": "https://cloud.google.com/translate/docs/reference/libraries",
+    "description":"A simple programmatic interface for detecting the language of arbitrary strings and translating them into dozens of languages using Neural Machine Translation.",
+    "transports": ["http"],
+    "icon":"https://cloud.google.com/_static/images/cloud/products/logos/svg/translate.svg",
```
Comment (briandealwis) [inline src/main/java/com/google/cloud/tools/libraries/libraries.json]: 404s

**Comment by patflynn** [comment]
@elharo should we not review this?

**Comment by elharo** [comment]
Branches got mixed up. Current version is at https://github.com/GoogleCloudPlatform/appengine-plugins-core/pull/449

**Pull request closed without merging**